src/ts/components/subtitlesettings/fontopacityselectbox.ts

```diff
@@ -1,37 +1,41 @@
-import {SubtitleSettingSelectBoxConfig, SubtitleSettingSelectBox} from './subtitlesettingselectbox';
+import {SubtitleSettingSelectBox} from './subtitlesettingselectbox';
 import {UIInstanceManager} from '../../uimanager';
-import {StorageUtils} from '../../storageutils';
-import {ColorUtils} from '../../colorutils';
 
 /**
  * A select box providing a selection of different font colors.
  */
 export class FontOpacitySelectBox extends SubtitleSettingSelectBox {
 
-  constructor(config: SubtitleSettingSelectBoxConfig) {
-    super(config);
-  }
-
   configure(player: bitmovin.PlayerAPI, uimanager: UIInstanceManager): void {
     super.configure(player, uimanager);
 
-    this.addItem('0.25', '25%');
-    this.addItem('0.50', '50%');
-    this.addItem('0.75', '75%');
-    this.addItem('1', '100%');
+    this.addItem(null, 'default');
+    this.addItem('100', '100%');
+    this.addItem('75', '75%');
+    this.addItem('50', '50%');
+    this.addItem('25', '25%');
 
-    this.selectItem('1');
+    this.onItemSelected.subscribe((sender, key: string) => {
+      this.settingsManager.fontOpacity.value = key;
 
-    if (StorageUtils.hasLocalStorage()) {
-      let color = window.localStorage.getItem('fontColor');
-      if (color != null) {
-        let col = ColorUtils.colorFromCss(color, ColorUtils.foreground);
-        this.selectItem(col.a.toString());
+      // Color and opacity go together, so we need to...
```

Comment:
Well by a default opacity of 0% I meant that in default setting there is no background which is kind of equivalent to 0% opacity. Once a background is set it goes to 100%, but if the user click on default it's not shown anymore, which is equivalent to clicking 0%. Unless there is a style tag settings different values.

The usecase that I don't like, is 

1. The user activate subtitles
2. He changes the opacity
3. Then the color
4. Actually prefer the "normal" opacity so click on default

=> color resets to white.

But I understand there is no easy workaround, so I guess for now it's ok.